### ISSUE TITLE
Request: insert time button

### ISSUE THREAD

**Issue by mthome** [baumgarr/Nixnote2#452]
There is an existing "Insert date and time button".  My workflow would be more efficient if I had an "insert time" (using current locale settings) option.  For completeness a button "insert date" would make sense.

An alternative would be to provide an "insert timestamp" button with a preference-settable format that could include date and/or time.

**Comment by robert7** [comment]
I added this function in [my fork in version 2.1-beta1](https://github.com/robert7/nixnote2/releases) (unfortunately maintainer of original version is currently not available).

I felt that adding 2 more buttons in toolbar would be a bit overkill, I don't want it too big, but I added 2 more keyboard shortcuts (keyboard combination is customizable) and tooltip info about 2 more options in existing button.

![screen_20180804_02](https://user-images.githubusercontent.com/1135218/43679984-3d5c16a2-9830-11e8-833f-a25ee8970b09.png)

I also added few more date formats (used mainly in european countries).

**Comment by mthome** [comment]
You might consider instead adding a default-disabled button - after all, you can turn the buttons on and off by preference... I had presumed that one could reorder them as well, but it seems not... at least not in the version I'm using.

**Comment by robert7** [comment]
Yes of course 2 more buttons could be added and hidden by default (rearranging in toolbar currently not supported), but the solution I selected was easier to implement. And this is the key, as my time for this project is very limited, so I need to prioritize.

So either its compromise, or you learn the keyboard shortcuts and will be faster anyway as by clicking on a toolbar - so it may be a win-win.
